skills/clean-user-facing-text/SKILL.md

```
@@ -15,23 +15,48 @@ Apply a final text-hygiene pass to prose the user owns or is authorized to proce
    - commands, paths, URLs, identifiers, API names, and exact values
    - formulas, citations, and text the user asks to quote verbatim
 3. Preserve every claim, fact, number, name, citation, and requirement.
-4. Establish the writing brief before changing prose:
+4. **Measure before.** Score the input with the vendored zero-LLM stylometry estimator (see Scoring) and record the score:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)